beanstalk_worker/worker_start_build.py

```diff
@@ -128,9 +128,15 @@ def start_build(job_details):
 while True:
     try:
         debug_log("listening to start_build tube")
-        job = bs.reserve()
-        job_details = json.loads(job.body)
-        result = start_build(job_details)
-        job.delete()
+        current_jobs_in_tube = int(dict(item.split(":") for item in bs.stats_tube('start_build').split('\n')[1:-1])['current-jobs-ready'])
```

Comment:
Converting a `dict` to `int`. I'm unclear as to how this is going to work. Can you provide a sample value/output of `bs.stats_tube`?